nixos/modules/services/networking/shairport-sync.nix

```diff
@@ -29,9 +30,44 @@ in
 
       package = lib.options.mkPackageOption pkgs "shairport-sync" { };
 
+      config =  mkOption {
+        type = configFormat.type;
+        default = {
```

Comment:
This should be moved to config otherwise we overwrite it completely when setting the option